step-templates/file-system-create-file.json

```diff
@@ -1,52 +1,59 @@
 {
   "Id": "175a91a9-562e-49b9-bfa6-609a4e16bc56",
   "Name": "File System - Create File",
-  "Description": "Creates a file, using the full path that is provided.",
+  "Description": "Creates a file under specified directory and write given content.",
   "ActionType": "Octopus.Script",
-  "Version": 2,
+  "Version": 3,
+  "CommunityActionTemplateId": null,
   "Properties": {
-    "Octopus.Action.Script.ScriptBody": "$filePath = $OctopusParameters['FilePath']\n$fileContent = $OctopusParameters['FileContent']\n$encoding = $OctopusParameters['Encoding']\n\nNew-Item -ItemType file -Path $filePath -Value '' -force\n\nif(![string]::IsNullOrEmpty($fileContent))\n{\n  Set-Content -path $filePath -value $fileContent -encoding $encoding\n}",
-    "Octopus.Action.Script.Syntax": "PowerShell"
+    "Octopus.Action.Script.ScriptBody": "$fileName = $OctopusParameters['FileName']\n$path = $OctopusParameters['Path']\n$content = $OctopusParameters['Content']\n\nif ($fileName -and $path) {\n\n if(-Not(Test-Path $path))\n    {\n       Write-Error \"$path does not exist\"\n       exit 1\n    }\n\n    $fullPath=Join-Path $path -ChildPath $fileName\n    \n    Write-Host \"File Path: $fullPath\"\n    \n    New-Item -type file -Path $fullPath -force\n\n    if($content){\n       Add-Content $fullPath $content\n    }\n\n    Write-Host \"$fileName created at $path\"\n}\nelse{\n    Write-Error \"File creation failed : Incorrect File name/path.\"\n    exit 1\n}\n",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
   },
-  "SensitiveProperties": {},
   "Parameters": [
     {
-      "Name": "FilePath",
-      "Label": "The full path where the file will be created.",
-      "HelpText": "Provide the entire physical path to which the physical file will be created. For example, 'C:\\Temp\\MyFile.txt' will create am empty text file named 'MyFile.txt' in the 'Temp' folder on the C: drive.",
-      "DefaultValue": null,
+      "Id": "607c3a21-a0ab-4b0d-aa01-0864bf2ae909",
+      "Name": "FileName",
```

Comment:
I'd suggest to keep the `FilePath` (instead of renaming it `Path`) and `FileContent` (instead of renaming it `Content`)  property names to minimize the impact to users upgrading their copies of this step template.